workflows/utilities/wf_read_QC_trim_ont.wdl

```diff
@@ -72,6 +76,22 @@ workflow read_QC_trim_ont {
     } 
   }
   if ("~{workflow_series}" == "theiaprok") {
+    if (call_kraken) {
```

Comment:
should we also add a `defined(kraken_db)` check here to make sure we don't get failures? I think people might try to run kraken by only setting `call_kraken` to true but not realizing you need to add a database?

Reply:
And if it's not defined, we can add the "kraken database not defined" message to the select_first if people want to know what's going on?

Reply:
That's an excellent suggestion that can be added to the illumina_pe and se read_qc_trim subworkflows as well!